src/Apps/Search/FilterState.tsx

```diff
@@ -157,6 +157,24 @@ export class FilterState extends Container<State> {
     )
   }
 
+  get hasFilters() {
+    return Object.keys(this.state).some(filter => {
```

Comment:
I think this currently covers all the filters logically, but I think this section could be more clear & less likely to change if we reused the `isDefaultFilter` helper from above.

For example, something like:

```
  get hasFilters() {
    return Object.entries(this.state)
      .some(([key, value]) => {
        return !isDefaultFilter(key, value)  
      })
  }
```

As the `hasFilters` function currently stands, if we add more filters/remove one, we need to make sure it's covered correctly both here and in `isDefaultFilter`. With the rewrite suggested, we would only have to make sure it's covered in `isDefaultFilter`.

Sorry I didn't catch this sooner, but looking at it made me wonder if all those clauses covered everything correctly. (Side note - that probably means it's worth writing tests for.)